zerver/management/commands/create_user.py

```diff
@@ -78,7 +83,8 @@ def handle(self, *args, **options):
                 full_name = input("Full name: ")
 
         try:
-            notify_new_user(do_create_user(email, initial_password(email),
+            pw = options.get('password', initial_password(email))
+            notify_new_user(do_create_user(email, pw,
```

Comment:
I had guessed the reason why this hasn't been implemented yet was due to such concern. Should the pw argument be passed as a file to be read instead?